pkg/otlp/collector.go

```diff
@@ -173,6 +173,7 @@ func NewPipeline(cfg PipelineConfig, s serializer.MetricSerializer) (*Pipeline,
 		DisableGracefulShutdown: true,
 		ConfigProvider:          configProvider,
 		LoggingOptions:          options,
+		SkipSettingGRPCLogger:   true,
```

Comment:
See comment on commit https://github.com/DataDog/datadog-agent/pull/15321/commits/3f4a78e5f2e276c8cdd90fa7e60455a2374d41d0 for more details on this change.